src/hub_predtimechart/hub_config.py

```diff
@@ -37,17 +37,17 @@ def __init__(self, hub_dir: Path):
             self.reference_date_col_name = ptc_config['reference_date_col_name']
             self.target_date_col_name = ptc_config['target_date_col_name']
             self.horizon_col_name = ptc_config['horizon_col_name']
+            self.initial_checked_models = ptc_config['initial_checked_models']
+            self.disclaimer = ptc_config['disclaimer']
 
         # set model_ids
-        model_ids = []
-        # for model_metadata_file in (hub_dir / 'model-metadata').glob('*.yml'):
+        self.model_id_to_metadata = {}
         for model_metadata_file in (list((hub_dir / 'model-metadata').glob('*.yml')) +
                                     list((hub_dir / 'model-metadata').glob('*.yaml'))):
             with open(model_metadata_file, 'r') as fp:
-                ptc_config = yaml.safe_load(fp)
-                model_id = f"{ptc_config['team_abbr']}-{ptc_config['model_abbr']}"
-                model_ids.append(model_id)
-        self.model_ids = sorted(model_ids)
+                model_metadata = yaml.safe_load(fp)
+                model_id = f"{model_metadata['team_abbr']}-{model_metadata['model_abbr']}"
```

Comment:
Are we assuming that this will parse as expected because teams' model metadata files have been validated upstream, or do we need to anticipate funny business?

Reply:
Good question. I'm assuming the hub files are all validated at this point.